packages/assets/scss/03-organisms/_utility-nav.scss

```diff
@@ -202,14 +202,18 @@ $utility-nav-height: 43px;
 
       @media ($bp-header-toggle-max) {
 
-        @include ma-accordion-icon;
-        display: block;
-        width: 100%;
+        & span {
 
-        &:after {
+          @include ma-accordion-icon;
+          // display: block;
```

Comment:
Yes, matching the structure of the other item that has "+", but screen readers don't announce the symbol although the Google Translate has `<a>` instead of `<button>`.
```
<li class="ma__utility-nav__item">
  <button class="ma__utility-nav__link js-util-nav-toggle" aria-haspopup="true" aria-label="Log in links for this page" aria-expanded="false">
    <svg aria-hidden="true" focusable="false">...</svg>
    <span>Log in to...</span>

    <!-- This .toggle-indicator has ::after that contains "+" -->
    <span class="toggle-indicator" aria-hidden="true"></span>
  </button>
  ...
</li>
```